time timeStamp
float64[] velocities